sdk/ml/azure-ai-ml/azure/ai/ml/operations/_job_operations.py

```diff
@@ -769,7 +769,7 @@ def _get_named_output_uri(
             try:
                 job = self.get(job_name)
                 artifact_store_uri = job.outputs[DEFAULT_ARTIFACT_STORE_OUTPUT_NAME]
-                if artifact_store_uri and artifact_store_uri.path:
+                if artifact_store_uri is not None and artifact_store_uri.path:
```

Comment:
`artifact_store_uri` got recognized as expression op because standalone job's outputs becomes `NodeOutput`, right?

Reply:
Maybe we can also investigate if we can just use `Output` instead of `NodeOutput` in standalone job.

Reply:
Yes, and I think it's worthy to do so as it can avoid some unexpected behavior and error